.github/workflows/windows.yml

```diff
@@ -16,13 +16,18 @@ jobs:
         double: [0, 1]
 
     steps:
-    - name: Build Gwion
-      uses: fennecdjay/gwion-action@v1
-      with:
-        dir: .
-        ref: ${{ github.sha }}
-        run: true
+    - name Checkout
+      uses: actions/checkout@v2
+
+    - name: Init submodules
+      run: git submodule update --init util ast
+
+    - name: Build
+      run: make
       env:
-        CC: gcc
+        CC: ${{ matrix.cc }}
```

Comment:
I afraid `${{ matrix.cc }}` will not be set in windows builds. Would you mind setting it back to `gcc`?